datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
This spot I removed already. But I need the spot where there is an instance of the superdataset.